keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/PausedRepository.kt

```diff
@@ -17,24 +17,27 @@
  */
 package com.netflix.spinnaker.keel.persistence
 
+import com.netflix.spinnaker.keel.pause.Pause
+import com.netflix.spinnaker.keel.pause.PauseScope
+import java.time.Clock
+
 /**
  * A repository to track what scopes are paused, starting with application
  */
 interface PausedRepository {
+  val clock: Clock
+    get() = Clock.systemUTC()
+
+  fun getPause(scope: PauseScope, name: String): Pause?
 
-  fun pauseApplication(application: String)
+  fun pauseApplication(application: String, user: String)
```

Comment:
Pause functions now take a user as an argument so we can record that in the database.